webapp/cas/main.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"io/ioutil"
```

Comment:
Please don't use tabs as indention, but 2 spaces. I'm surprised `gofmt` lets this pass...

Reply:
From the [Effective go documentation](https://golang.org/doc/effective_go.html#formatting)
> Indentation
We use tabs for indentation and gofmt emits them by default. Use spaces only if you must.